<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>900</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1600</width>
    <height>900</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#Widget  {
 
	background-color: rgb(37, 40, 48);

}

QWidget#widget_input_select  {
 
		 border: 4px ridge rgb(37, 40, 48); 

}



QLabel#lb_title {
    color: #ffffff; 
}
QLabel#lb_img {
   
	  border: 4px ridge rgb(37, 40, 48); 
}

QPushButton {
    color: white;  /* 使用颜色名称 */
 	border:none;
	padding: 0px;
    margin: 0px;
	
}

QPushButton#btn_start_cam {
    color: white;  
	border: 3px outset rgb(60, 65, 75);  
    
    border-radius: 10px;
	
}

QPushButton#btn_stop_cam {
    color: white;  
	border: 3px outset rgb(60, 65, 75);  
    
    border-radius: 10px;
	
}
QPushButton:hover {
    background-color: #2980b9;  /* 悬停时 - 深蓝色 */
}
QPushButton:checked {
    background-color: #2980b9;
}
QFrame#frame {
	 background-color:rgb(102,102,102);  /* 颜色 */
    max-height: 1px;            /* 高度1像素 */
    min-height: 1px;
    border: none;

}
QFrame#frame_2 {
	 background-color:rgb(100,100,100);  /* 颜色 */
    max-width: 1px;            /* 高度1像素 */
    min-width: 1px;
    border: none;
         

}

QComboBox {
    color: white;
    background-color: rgb(37, 40, 48);
}

QComboBox QComboBoxListView {
    background-color: rgb(37, 40, 48);
    border: 1px solid rgb(80, 85, 95);
    outline: 0px;
}

QComboBox QComboBoxListView::item:hover,
QComboBox QComboBoxListView::item:selected {
    background-color: #2980b9;
    color: white;
}</string>
  </property>
  <widget class="QLabel" name="lb_title">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>40</y>
     <width>721</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>40</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>YOLO智能视频监测平台</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>240</y>
     <width>1271</width>
     <height>482</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="lb_img">
      <property name="minimumSize">
       <size>
        <width>640</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>640</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QWidget" name="widget_input_select" native="true">
      <property name="minimumSize">
       <size>
        <width>577</width>
        <height>480</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>577</width>
        <height>480</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QPushButton" name="btn_local">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>48</height>
              </size>
             </property>
             <property name="text">
              <string>本地摄像头</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QFrame" name="frame_2">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>1</width>
               <height>48</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="frameShape">
              <enum>QFrame::VLine</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Sunken</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_remote">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>48</height>
              </size>
             </property>
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="text">
              <string>远程摄像头</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QFrame" name="frame">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>578</width>
             <height>1</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::HLine</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Sunken</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QStackedWidget" name="stackedWidget">
           <property name="mouseTracking">
            <bool>true</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel{
	color:white;
}

QPushButton {
	border: 3px outset rgb(60, 65, 75);  
    
    border-radius: 5px;
}



QLineEdit{
	color:white;
	
	background-color: rgb(37, 40, 48);

}
</string>
           </property>
           <property name="currentIndex">
            <number>1</number>
           </property>
           <widget class="QWidget" name="page_local">
            <widget class="QWidget" name="layoutWidget">
             <property name="geometry">
              <rect>
               <x>170</x>
               <y>110</y>
               <width>201</width>
               <height>161</height>
              </rect>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QPushButton" name="btn_start_cam">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>50</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <pointsize>16</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>启动摄像头</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
           <widget class="QWidget" name="page_remote">
            <widget class="QWidget" name="layoutWidget">
             <property name="geometry">
              <rect>
               <x>82</x>
               <y>82</y>
               <width>351</width>
               <height>151</height>
              </rect>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_4">
                <item>
                 <widget class="QLabel" name="label">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>35</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>地址</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_2">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>40</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>协议</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_3">
                <item>
                 <widget class="QLineEdit" name="le_addr">
                  <property name="text">
                   <string>192.168.1.19:8554/live</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="comboBox">
                  <property name="mouseTracking">
                   <bool>true</bool>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">QComboBox {
    color: white;
    background-color: rgb(37, 40, 48);
}

/* 下拉弹出列表整体背景 */
QComboBox QListView {
	color:white;
    background-color: rgb(37, 40, 48);
    border: 1px solid rgb(80, 85, 95);
    outline: 0px;
}

/* 悬停变蓝 */
QComboBox QListView::item:hover {
    background-color: #2980b9;
    color: white;
}

/* 选中也变蓝 */
QComboBox QListView::item:selected {
    background-color: #2980b9;
    color: white;
}</string>
                  </property>
                  <item>
                   <property name="text">
                    <string>RTSP</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>RTMP</string>
                   </property>
                  </item>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="layoutWidget">
             <property name="geometry">
              <rect>
               <x>150</x>
               <y>240</y>
               <width>216</width>
               <height>45</height>
              </rect>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QPushButton" name="btn_remote_conn">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>104</width>
                  <height>43</height>
                 </size>
                </property>
                <property name="text">
                 <string>连接</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
